libcrux-ml-dsa/src/ntt.rs

```diff
@@ -94,6 +94,7 @@ fn invert_ntt_at_layer_2<SIMDUnit: Operations>(
             ZETAS_TIMES_MONTGOMERY_R[*zeta_i],
         );
     }
+    ()
```

Comment:
Please add comments to these so that we can undo this once the issue is fixed.

Reply:
Done